src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, Color color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rightTriangle,"
+                + "area: " + Math.floor(getArea()) + " sq.units, "
+                + "firstLeg: " + firstLeg + " units"
+                + ", secondLeg: " + secondLeg + " units"
+                + ", color: " + color.toString().toLowerCase());

```

Comment:
[to get a string representation of an enum use method `name()`](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants) Fix it in all `draw()` methods.